www/.nvmrc

```diff
@@ -0,0 +1 @@
+0.10.33
```

Comment:
Do we think we should use 0.12+ for future projects? It introduces the new Streams 3 API which is meant to be more performant, especially since gulp is nothing but streams.